todo_app/templates/index.html

```diff
@@ -11,14 +11,14 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Your To-Dos</h2>
       <ul class="list-group mb-4" style="list-style: none;">
-        {% for to_do in to_dos -%}
-            <li>{{ to_do.status }}: {{ to_do.title }}</li>
-            <form action="/complete/{{to_do.id}}" method="post">
+        {% for item in view_model.items -%}
```

Comment:
This fits nicely with the way you handle it, but worth consciously noting that in choosing to use `items` here, the `done_items` etc. properties are very much a waste of code, and just exist for the sake of our unit tests - but since they do prove that point that's not a problem here!

Reply:
This was quite a large oversight, I was intending to use the properties in the actual template! Have now implemented it properly, verified that the tests failed (as per one of your other comments) and corrected. A valuable lesson in checking 😆